app/Console/Commands/InstallCommand.php

```diff
@@ -298,6 +303,33 @@ protected function configureCachet()
     }
 
     /**
+     * Configure the first user.
+     *
+     * @return void
+     */
+    protected function configureUser()
+    {
+        if (!$this->confirm('Do you want to create an admin user?')) {
+            return;
+        }
+
+        // We need to refresh the config to get access to the newly connected database.
+        $this->getFreshConfiguration();
+
+        // Now we need to install the application.
+        $this->call('app:install');
```

Comment:
I wonder if it is really worth cachet loading in my laravel core package. Maybe it should just define it's own app commands, to replace the originals?